s2a_options.go

```diff
@@ -121,6 +125,34 @@ type ClientOptions struct {
 	// VerificationMode specifies the mode that S2A must use to verify the
 	// peer certificate chain.
 	VerificationMode VerificationModeType
+
+	// S2Av2 only.
+	// Optional fallback after dialing with S2Av2 fails.
+	FallbackOpts *FallbackOptions
+}
+
+// FallbackOptions contains the fallback options for S2Av2.
```

Comment:
I think we need to say a bit more here, e.g. "FallbackOptions prescribes the fallback logic that should be taken if the application fails to connect to S2A."